modules/installation-osp-kuryr-octavia-configuration.adoc

```diff
@@ -110,12 +110,12 @@ your specific installation of {rh-openstack}. See the {rh-openstack}
 documentation for further information. For more information on customizing your
 Octavia installation, see
 https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/13/html-single/networking_guide/#planning_your_octavia_deployment[installation
-of Octavia using Director].
+of Octavia using director].
```

Comment:
I don't know how OSP is using capitalization, but _Director_ is probably a Specific Thing. I don't know if it makes sense to lowercase it?